.github/workflows/build-test-lint.yml

```diff
@@ -0,0 +1,81 @@
+name: Node Build / Test / Lint
+
+# Run this when someone pushes a branch
+on: [push]
+
+jobs:
+  # Get the versions of node and npm as defined in the package.json engines
+  get-versions:
+    # Running on Lob's GitHub runners
+    runs-on: ubuntu-latest
+
+    outputs:
+      node_version: ${{ steps.node_version.outputs.prop }}
+      npm_version: ${{ steps.npm_version.outputs.prop }}
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Read node version from package.json
+      id: node_version
+      uses: notiz-dev/github-action-json-property@v0.1.0
+      with:
+        path: "package.json"
+        prop_path: "engines.node"
+
+    - run: |
+        echo "Node version found: ${{ steps.node_version.outputs.prop }}"
+
+    - name: Read npm version from package.json
+      id: npm_version
+      uses: notiz-dev/github-action-json-property@v0.1.0
+      with:
+        path: "package.json"
+        prop_path: "engines.npm"
+
+    - run: |
+        echo "npm version found: ${{ steps.npm_version.outputs.prop }}"
+
+  # Run lint and tests
+  run-tests:
+    # Running on Lob's GitHub runners
```

Comment:
Non-blocking, but you're using the `ubuntu-latest` runner (throughout both files) which is actually the GitHub runner. Would you mind updating the comments?

Reply:
oh, yes, good catch, thanks!